src/JuliusSweetland.OptiKey/UI/ValueConverters/IsActionKey.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using JuliusSweetland.OptiKey.Models;
+
+namespace JuliusSweetland.OptiKey.UI.ValueConverters
+{
+    public class IsActionKey : IValueConverter
```

Comment:
Could these new Key specific converters be renamed to include a reference to the fact that the key is a CommuniKate key